### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by johnpryan**

### CODE REVIEW

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -0,0 +1,250 @@
+# Created with package:mono_repo v3.1.0-beta.2-dev
```
Comment (kevmoo) [inline .github/workflows/dart.yml:1]: Dude! Install the latest!

Comment (kevmoo) [reply .github/workflows/dart.yml:1]: 3.4.5 is out.

Comment (johnpryan) [reply .github/workflows/dart.yml]: Done

Comment (johnpryan) [reply .github/workflows/dart.yml]: This branch is verrry old.

Comment (kevmoo) [reply .github/workflows/dart.yml]: Uh huh!

File: mono_repo.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
+# Run `mono_repo generate --validate` to check 
+# that everything is up to date
+self_validate: true
+github:
 travis:
```
Comment (kevmoo) [inline mono_repo.yaml:5]: If you want to *migrated* to GitHub actions, delete this line and delete `.travis.yml`

File: mono_repo.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
+# Run `mono_repo generate --validate` to check 
+# that everything is up to date
+self_validate: true
+github:
```
Comment (kevmoo) [inline mono_repo.yaml:4]: Add `cron` – good to run these weekly and catch regressions w/ the SDK dev branch

https://github.com/google/mono_repo.dart/blob/master/mono_repo.yaml#L8

File: mono_repo.yaml (hunk 2 above)
Comment (johnpryan) [reply mono_repo.yaml]: sgtm, done.

**Comment by kevmoo** [comment]
Don't land this!

Gah. I messed up configuration. Need to roll a new mono_repo

...thankfully, looking at this helped me find it!

**Comment by kevmoo** [comment]
Okay. install `3.4.6` and try again. Sorry!

**Comment by johnpryan** [comment]
@kevmoo what was the change in 3.4.6? I don't see a difference in the generated output.

**Comment by kevmoo** [comment]
> @kevmoo what was the change in 3.4.6? I don't see a difference in the generated output.

Ha! turns out...uh...nevermind

Ugh. Sorry

**Comment by kevmoo** [comment]
...I *did* fix a bug. Just not affecting you. I swear... 🤷

**Review by kevmoo — APPROVED** [review]
Just remove the `(Draft)` bits from the title! Thanks!

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Dart samples
 
-[![Build Status](https://travis-ci.org/dart-lang/samples.svg?branch=master)](https://travis-ci.org/dart-lang/samples)
+![example workflow name](https://github.com/dart-lang/samples/workflows/Dart%20CI/badge.svg)
```
Comment (kevmoo) [inline README.md:3]: Gah! one more thing! Make this link to the action?

Comment (johnpryan) [reply README.md]: The best link I could find to use was https://github.com/google/mono_repo.dart/actions?query=branch%3Amaster  - the [actions page](https://github.com/google/mono_repo.dart/actions) uses search syntax 🤷

Comment (kevmoo) [reply README.md]: LGTM!

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**